pkgs/racket-doc/scribblings/reference/stx-ops.scrbl

```diff
@@ -488,5 +488,15 @@ limited to---the following keys:
 
 @history[#:added "6.3"]}
 
+
+@defproc[(syntax-mapped-symbols [stx syntax?])
+         (listof (cons/c (or/c exact-integer? #f) (listof symbol?)))]{
+Return a association list mapping @tech{phase level} values(where
+@racket[#f] corresponds to the @tech{label phase level}) to names
```

Comment:
Should names be symbols?

Reply:
I am not sure, but I will change it to "symbols" since you think it is better.